### DISCUSSION TITLE
Upgrade librsvg to 2.5

### DISCUSSION THREAD

**Discussion by yisibl** [libvips/libvips#2496] [Q&A]
The new version supports more SVG features and the maintainer of librsvg said: 

> Please do not use librsvg 2.40.x; it cannot render recent Adwaita icon themes correctly.
>
> The librsvg 2.40.x series is the last "C only" version of the library; it was deprecated in 2017.

https://people.gnome.org/~federico/blog/do-not-use-librsvg-2.40.x.html

**Comment by jcupitt** [comment]
Hi @yisibl,

I think we're using something between 2.50 and 2.52 at the moment for testing and deplyment. If your libvips is built against an older librsvg, it's probably an issue with whoever packaged it for you.

**Comment by yisibl** [comment]
Where can I see what version of librsvg is currently in use?

**Reply by jcupitt** [reply]
It depends on your platform. What OS are you using?

**Reply by yisibl** [reply]
macOS 10.14.5,  I found "libvips" is "8.11.3"

**Reply by jcupitt** [reply]
Then you need `brew info librsvg`. I see 2.50.7.

**Reply by yisibl** [reply]
Thanks!
